examples/undocumented/python/graphical/regression_gaussian_process_modelselection.py

```diff
@@ -26,7 +25,7 @@ def regression_gaussian_process_modelselection (n=100, n_test=100, \
 	feats_test = RealFeatures(X_test)
 
 	# GP specification
-	kernel = GaussianKernel(10, 0.05)
+	kernel = kernel("GaussianKernel", log_width=log(0.05))
```

Comment:
is log defined?

Reply:
yes there is np.* :)

Reply:
mmmmh....
generally I would prefer `import shogun as sg` and `import numpy as np`
But dont worry about it in the old school examples